Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -275,6 +275,47 @@ def check_python_symlinks(name, keg_only)
     "Python formulae that are keg-only should not create `pip3` and `wheel3` symlinks."
   end
 
+  INVALID_EMPTY_FILE_NAMES = %w[AUTHORS ChangeLog changes COPYING NEWS README TODO].freeze
+  INVALID_EMPTY_FILE_EXTENSIONS = %w[.html .md .rst .txt].freeze
+  INVALID_EMPTY_FILE_LOCATIONS = %w[bin libexec share/man share/doc].freeze
+  # Names and extensions to ignore, even if they are in a location within prefix given above
+  VALID_EMPTY_FILE_NAMES = %w[__init__.py REQUESTED].freeze
```

Comment:
```suggestion
  VALID_EMPTY_FILE_NAMES = %w[__init__.py empty REQUESTED].freeze
```

Found with `find /usr/local/Cellar -type f -empty | xargs basename | sort | uniq` (in case it's useful)